conference/cart.py

```diff
@@ -38,7 +37,7 @@
 
 
 GLOBAL_MAX_PER_FARE_TYPE = 6
-ORDER_CONFIRMATION_EMAIL_SUBJECT = "EuroPython2019: Order confirmation"
+ORDER_CONFIRMATION_EMAIL_SUBJECT = "EuroPython2020: Order confirmation"
```

Comment:
Should we create a ticket to replace these with a settings variable or a constant?

Reply:
That's a good idea - although I think the first step to cleaning up the mailing would be putting it all in a single module; right now it would require quite some searching to find out all templates that would need to be updated.

Reply:
Agree, we should probably focus on cleaning and consolidating the codebase, then we can search and replace where needed.